docs/Validation.md

```diff
@@ -383,7 +384,9 @@ Server-side validation is supported out of the box for `pessimistic` mode only.
 
 **Tip**: The returned validation errors might have any validation format we support (simple strings, translation strings or translation objects with a `message` attribute) for each key.
 
-**Tip**: If your data provider leverages React Admin's [`httpClient`](https://marmelab.com/react-admin/DataProviderWriting.html#example-rest-implementation), all error response bodies are wrapped and thrown as `HttpError`. This means your API only needs to return an invalid response with a json body containing the `errors` key.
+Note: you can use the `message` key to customize the message displayed in the notification popover but it is not mandatory. By default the notification message is the translatable key `ra.notification.http_error`.
```

Comment:
I thought this only worked with React 17. Should we really mention this in the docs then?